frontend/plan/components/Cart.tsx

```diff
@@ -1,10 +1,23 @@
 import React from "react";
 import { connect } from "react-redux";
-import PropTypes from "prop-types";
 import CartSection from "./CartSection";
 import { meetingsContainSection, meetingSetsIntersect } from "./meetUtil";
 import { removeCartItem, toggleCheck, fetchCourseDetails } from "../actions";
 
+import { ThunkDispatch } from "redux-thunk";
+import { Section, CartCourse } from "../types";
+
+interface CartProps {
+    courses: CartCourse[];
+    toggleCourse: (courseId: Section) => void;
```

Comment:
would rename this from `courseId` to `section` in the signature to be more specific. There's definitely a conflation between `course` and `section` in the code (we inherited that unfortunately), would be good to start moving things over to be more specific.